src/main/java/com/amazon/ion/IonEncodingVersion.java

```diff
@@ -24,7 +24,7 @@ public abstract class IonEncodingVersion<BinaryWriterBuilder, TextWriterBuilder>
      * Ion 1.0, see the <a href="https://amazon-ion.github.io/ion-docs/docs/binary.html">binary</a> and
      * <a href="https://amazon-ion.github.io/ion-docs/docs/text.html">text</a> specification.
      */
-    public static IonEncodingVersion<IonBinaryWriterBuilder, IonTextWriterBuilder> ION_1_0 = new IonEncodingVersion<IonBinaryWriterBuilder, IonTextWriterBuilder>(0) {
+    public static final IonEncodingVersion<IonBinaryWriterBuilder, IonTextWriterBuilder> ION_1_0 = new IonEncodingVersion<IonBinaryWriterBuilder, IonTextWriterBuilder>(0) {
```

Comment:
I'm not sure why Spotless just found these now, but this is a good change, though unrelated to the rest of the PR.